.github/CONTRIBUTING.md

```diff
@@ -25,9 +25,8 @@ and let everybody know that you intend to work on it.
    the [Discourse](https://discourse.bokeh.org) for similar or identical issues. Keyword
    searches for your error messages are usually effective.
 
-2. The issue may already be resolved. Always try to reproduce the issue using most recent
-   [development build](http://bokeh.pydata.org/docs/installation.html#developer-builds) of
-   Bokeh, or even better, by building and installed from the `master` branch.
+2. The issue may already be resolved. Always try to reproduce the issue by building and
+   installed from the `master` branch.
```

Comment:
So the reason that the dev build install instructions were removed was that, despite the fact that we were and always have been extremely clear that dev build JS resources ***are not permanent*** that dev builds should NEVER be used "for production", a specific users leveled a very unpleasant interaction at us when old BokehJS dev builds did not get migrated to the new CDN. Rather than put up with that grief, I removed the dev build section from the prominent "Installation" section of the docs. However, I could be persuaded that a section on installing dev builds should go in the *Dev Guide* (along with very loud and very stern disclaimers). Would you be interested in re-instatiting the old dev build install instructions in the dev guide?

Reply:
I could see the benefit of providing dev builds to get feedback from users that don't have the time or knowledge to build Bokeh themselves, but I am not familiar enough with the project or this specific incident to know what is the smart thing to do here.

So this file is specific for reporting bugs.. Since you seem to have a pretty active release cycle, it looks like bugfixes would reach the actual releases soon too. This would mean that it would not be that beneficial to recommend or mandate bug reporters to test the dev build first, as it would not contain many bugfixes that are not in the latest release. And for the few false-positives, where you suspect the reported bug to be fixed in the dev build, you can always manually hand out the dev build to the reporter for testing.

But again: I'm not that familiar with this project and I think you're in a better position to decide this.

Reply:
Well the main thing is just that, given the need to build BokehJS, building/testing from master is a pretty large hurdle for most people. I'm mostly indifferent about suggesting dev builds here but if we leave it out I'd suggest maybe just simplifying to:

> Always try to reproduce the issue n the latest stable release

What do you think?

Reply:
Sounds good to me!